adage/analyze/tests.py

```diff
@@ -625,19 +642,21 @@ def test_activity_mlmodel_filter(self):
         """
         Test "activity/?sample=<id>&mlmodel=<ml_id>" API.
         """
-        uri = (self.baseURI + "activity/?sample=" +
-               str(self.random_object(Sample).id) + "&mlmodel=" +
-               str(self.random_object(MLModel).id))
+        uri = self.baseURI + "activity/"
+        data = {
```

Comment:
Right - I agree, we should always do this. Also it makes sense that it was breaking the tests because of the strange characters that fixtureless produced were being manually concatenated onto the url string right?